helpers/role_helper.py

```diff
@@ -1,30 +1,35 @@
 import discord.ext.commands
-from discord_slash.utils.manage_components import create_button, create_actionrow
+from discord_slash.utils.manage_components import create_button, create_actionrow, create_select, create_select_option
 from discord_slash.model import ButtonStyle
 from services import dm_service
 from asyncio import Lock
 import time
 
 programme_roles_dict = {
-    'cse': 'Computer Science and Engineering',
-    'ae': 'Aerospace Engineering',
-    'mech': 'Mechanical Engineering',
-    'ee': 'Electrical Engineering',
-    'nb': 'Nanobiology',
+    'cse': ['Computer Science and Engineering', '\U0001F4BB'],  # laptop
+    'ae': ['Aerospace Engineering', '\U0001F680'],  # rocket
+    'mech': ['Mechanical Engineering', '\U0001F527'],  # wrench
+    'ee': ['Electrical Engineering', '\U0001F39B'],  # control knobs
+    'nb': ['Nanobiology', '\U0001F9EC'],  # dna
+    'aes': ['Applied Earth Science', '\U0001FAA8'],  # rock
+    'ap': ['Applied Physics', '\U0000269B'],  # atom
+    'a': ['Architecture', '\U0001F3DB'],  # classical building
+    'che': ['Chemical Engineering', '\U0001F9EA'],  # test tube
+    'idp': ['IDP', '\U0001F615'],  # confused face no idea what idp is
+    'lst': ['Life Science & Technology', '\U0001F9EB'],  # petri dish
+    'mst': ['Molecular Science & Technology', '\U00002697'],  # alembic
+    'm': ['Mathematics', '\U0001F9EE'],  # abacus
+    'tpm': ['TPM (wannabe engineers)', '\U0001F9D1']  # factory worker
+    # 'ce': 'Civil Engineering'
 }
 
-programme_roles = set(programme_roles_dict.values())
+programme_roles_tud = ['cse', 'ae', 'mech', 'ee', 'nb', 'aes', 'ap', 'a', 'lst', 'mst', 'm', 'tpm']
+programme_roles_tue = ['cse', 'mech', 'ee', 'ap', 'a', 'che', 'm']
+programme_roles_utwente = ['cse', 'mech', 'ee', 'ap', 'che', 'm']
```

Comment:
We might want to do something about programmes that are called differently at different unis. For example, "Computer Science and Engineering" is called "Technical Computer Science" at UTwente, so people might be confused if they see "Computer Science and Engineering" under the UTwente dropdown.

Not a big issue though, so fixing this is not really worth making the code messier and less maintainable. I'll see if there's a simple way to fix it.